index.html

```diff
@@ -105,5 +107,219 @@ <h1 class="title">I’m Michael Glad to see you!</h1>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+
+      <div class="projects">
+        <img src="images/snapshoot-project-2.png" alt="snapshoot-project-2" />
+        <p class="project-title">Multi-Post Stories</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+
+      <div class="projects">
+        <img src="images/snapshoot-project-3.png" alt="snapshoot-project-3" />
+        <p class="project-title">Tonic</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+
+      <div class="projects">
+        <img src="images/snapshoot-project-4.png" alt="snapshoot-project-4" />
+        <p class="project-title">Multi-Post Stories</p>
+        <div class="project-details">
+          <span class="client-label">canopy</span>
+          <span class="middot">&middot;</span>
+          <span class="role-label">back end dev</span>
+          <span class="middot">&middot;</span>
+          <span class="year-label">2015</span>
+        </div>
+        <div class="project-desc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </div>
+        <ul class="tags">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <button type="button" class="primary-btn">See Project</button>
+      </div>
+    </section>
+    <section class="about-section">
+      <div class="top-section">
+        <p class="title">About Myself</p>
```

Comment:
- [ ] Same thing here about the title, it should be wrapped in a `hx` tag, kindly change that. 😄

Reply:
- Resolved the issue thank your for this feedback @mirouhml